bitnami/harbor/templates/portal/portal-dpl.yaml

```diff
@@ -60,10 +90,36 @@ spec:
             timeoutSeconds: {{ .Values.portal.readinessProbe.timeoutSeconds }}
             successThreshold: {{ .Values.portal.readinessProbe.successThreshold }}
             failureThreshold: {{ .Values.portal.readinessProbe.failureThreshold }}
+          {{- else if .Values.portal.customReadinessProbe }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.portal.customReadinessProbe "context" $) | nindent 12 }}
           {{- end }}
           env:
             - name: BITNAMI_DEBUG
               value: {{ ternary "true" "false" .Values.portalImage.debug | quote }}
+            {{- if .Values.portal.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.portal.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.portal.extraEnvVarsCM }}
+            - configMapRef:
```

Comment:
do not we need checksum for these `configMapRef` and `secretRef`?

Reply:
Same as before, we don't render it